drivers/linux/Makefile

```diff
@@ -46,10 +46,24 @@ KSYMS_MNIC = $(KMOD_OUT_DIR)/Module.symvers.mnic
 KSYMS_UIO = $(KMOD_OUT_DIR)/Module.symvers.uio
 
 DVER = $(shell cd $(KMOD_SRC_DIR) ; git describe --tags 2>/dev/null)
+ifneq ($(DVER),)
+    XX = $(shell echo $(DVER) > $(KMOD_SRC_DIR)/git_tag ; ls $(KMOD_SRC_DIR) )
+    $(info b1 DVER $(DVER) XX $(XX) )
+else
+    DVER = $(shell cat $(KMOD_SRC_DIR)/git_tag)
+    $(info b2 DVER $(DVER) )
+endif
 
 else
 
+# this host driver build assumes we're building in the linux-ionic directory
+# or from the drivers-linux-eth package ./drivers directory
+
 DVER = $(shell git describe --tags 2>/dev/null)
+ifeq ($(DVER),)
+    DVER = $(shell cat git_tag 2>/dev/null )
+    $(info b3 DVER $(DVER) )
+endif
```

Comment:
Do we really need this "tag" stuff in the dsc-drivers repo?

Reply:
> Do we really need this "tag" stuff in the dsc-drivers repo?

I was trying to decide how much more I needed to trim when moving it to dsc-drivers - but yes I think this could go away too.
I really don't want to create an automation for stripping rdma and pbapi stuff, but this is getting annoying.